src/engraving/layout/layoutoptions.h

```diff
@@ -40,6 +40,7 @@ struct LayoutOptions
     LayoutMode mode = LayoutMode::PAGE;
 
     bool isShowVBox = true;
+    double noteHeadWidth = 0.0;
```

Comment:
I don't think this should be in options. It is rather something similar to style. In fact, this value is fully derived from style settings, see `Score::spatiumChanged`. The only reason that it is a separate field, is for performance (caching). So maybe `LayoutConfiguration` itself is the best place for it.

Reply:
In a next PR I will see what can be done about it.